### ISSUE TITLE
508-defect-2 [COGNITION] Close this modal button SHOULD be descriptive and consistent

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#19794]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a screen reader user, I want to know if closing the modal will cancel my changes or allow me to continue editing.
As a sighted user, I want to confidently know that closing the modal will _not_ cancel my changes.

## Details

A modal appears asking the user if they are certain they want to cancel before moving forwards. Several issues may emerge:
- For screen reader users, the `aria-label` of the close icon will announce "Close this modal"-- but won't describe _what_ closing will do.
- Sighted users may become confused at how color is handled. The dark blue fill of the close icon mimics that of the "cancel" button, so users may assume closing the modal will also "cancel" the selection.

## Acceptance Criteria

- [ ] It is clear to users what each selection will do

## Environment

* Operating System: any
* Browser: any
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Enter `https://staging.va.gov/profile/direct-deposit` in browser
2. Start screenreading device listed in Environment
3. Edit the first field and click the "cancel" button

## Proposed Solution (if known)

In progress

## WCAG or Vendor Guidance (optional)

In progress

## Screenshots or Trace Logs

<img width="1792" alt="Screen Shot 2021-02-10 at 3 54 30 PM" src="https://user-images.githubusercontent.com/14154792/107572349-2859e080-6bba-11eb-88f4-a382a24fca4c.png">

**Comment by joshkimux** [comment]
Closing after review!
